ThingAppraiser/input/input_type/FileReader.cs

```diff
@@ -0,0 +1,38 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.IO;
+
+namespace ThingAppraiser.Input
+{
+    class FileReader
```

Comment:
As I mentioned previously, if you have stateless class, you can mark it as `static`.

```suggestion
    public static class FileReader
```